Update/some_002.txt

```diff
@@ -146,15 +146,15 @@ void main()
 //rおかーさんてば、朝から元気だなぁ…。なんてぼんやり思いながら、始まったら最後、終わることのないおかーさんの小言から逃れようと、蒲団の中でセーフティゾーンを探る。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "おかーさんてば、朝から元気だなぁ…。なんてぼんやり思いながら、始まったら最後、終わることのないおかーさんの小言から逃れようと、蒲団の中でセーフティゾーンを探る。",
-			NULL, "Mom, I'm fine this morning...  Vaguely thinking that, I realized that Mom was starting to go off on another of her never-ending rants, so I tried to escape to the safe zone in my futon.", Line_Normal);
+			NULL, "God mom, you're pretty energetic early in the morning...  Vaguely thinking that, I realized that Mom was starting to go off on another of her never-ending rants, so I tried to escape to the safe zone in my futon.", Line_Normal);
```

Comment:
```suggestion
			NULL, "God, Mom! You're pretty energetic early in the morning... Vaguely thinking that, I realized that Mom was starting to go off on another of her never-ending rants, so I tried to escape to the safe zone in my futon.", Line_Normal);
```

"Mom" is being used as a proper name in this case, so it gets capitalized. Added the "!" to prevent a string of commas and match tone.